mne/epochs.py

```diff
@@ -1420,6 +1420,11 @@ def next(self, return_event_id=False):
 
         return epoch if not return_event_id else epoch, self.event_id
 
+    @property
+    def times(self):
+        """Time vector in seconds."""
+        return self._times.copy()
```

Comment:
At one point we had a performance regression because of this sort of thing (tens of thousands of accesses to `epochs.times` was the bottleneck!). Could we just set the NumPy flag writeable false?

Reply:
(By doing so, `epochs.times[:] = ...` will also give an error, in the current code it will not error, but also not do what the user expects)

Reply:
good point

Reply:
done